js/mviewer.js

```diff
@@ -607,8 +607,13 @@ mviewer = (function () {
                         summary += '<a href="'+_overLayers[this.layer].metadata+'" target="_blank">En savoir plus</a>';
                     }
                     _overLayers[this.layer].summary = summary;
+
+                    var modifiedDate = $(result).find("dct\\:modified, modified").text() || $(result).find("dct\\:created, created").text();
```

Comment:
XPath to be double checked ...

Reply:
Looking here https://www.dublincore.org/specifications/dublin-core/dcmi-terms/ and here https://www.w3.org/TR/vocab-dcat-2/#Property:resource_update_date seem to be the correct fields (as far as I understand)

Reply:
Oh, you're requesting the result as dublin core, not ISO ? Then it's fine.
Could you paste an example of MD URL which gets queried ?

Reply:
Yep, here's the record for the lycée layer of the default.xml https://kartenn.region-bretagne.fr/geonetwork/srv/eng/csw?SERVICE=CSW&VERSION=2.0.2&REQUEST=GetRecordById&elementSetName=full&ID=99e78163-ce9a-4eee-9ea0-36afc2a53d25